addons/mission/functions/fnc_dialogue.sqf

```diff
@@ -0,0 +1,37 @@
+#include "script_component.hpp"
+/*
+ * Author: Mike
+ * On-screen text mimicking speech.
+ *
+ * Call only on the server, initServer or wrapped in if (isServer) then
+ *
+ * Arguments:
+ * 0: Person Speaking <STRING>
+ * 1: Message <STRING>
+ * 2: Colour of text <NUMBER> (optional - default: White)
+ * 3: On screen time <NUMBER> (optional - default: 2)
+ *
+ * Colours are:
+ * 0: White
+ * 1: Black
+ * 2: Blue
+ * 3: Red
+ * 4: Yellow
+ * 5: Green
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["John James", "I hate all of you.", 0, 3] call MFUNC(dialogue)
```

Comment:
I still prefer `MFUNC(say)` for some reason. It's literally a person saying something. There is no **di**alogue happening.